### PR TITLE
Adding Medal: MeDAL: Medical Abbreviation Disambiguation Dataset for Natural Language Understanding Pretraining

### PR DESCRIPTION
**Pull request by Narsil**

### CODE REVIEW

File: datasets/medal/README.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+---
+- YAML tags: copy-paste the tags obtained with the tagging app: http://34.68.228.168:8501/ or https://github.com/huggingface/datasets-tagging for local tagging
```
Comment (lhoestq) [inline datasets/medal/README.md:2]:
```suggestion
```

**Review by lhoestq — APPROVED** [review]
Thanks ! hopefully we won't have any issues with the url

**Comment by lhoestq** [comment]
Could you fix the dummy data before we merge ?
Looks like the dummy `train.csv` is missing

**Review by lhoestq — APPROVED** [review]
LGTM !

**Comment by xhlulu** [comment]
Thanks @Narsil @lhoestq for adding MeDAL :)

**Pull request merged**